### PR TITLE
[iOS] remove arbitrary framerate cap.

### PR DESCRIPTION
**Pull request by jonahwilliams**
This doesn't actually seem to be helping. For the most part, the frame time of the banners is steady with occassional spikes. Adding the framerate cap doesn't help the spikes, since those are in the 30+ ms frame time, and just results in everything else looking janky as well (especially if we dance around the cap, then we go 120 - 80)

Since this is based on an arbitrary number, from an arbitrary app, on an arbitrary point in time we should remove it and instead try to fix the performance problems

### CODE REVIEW

**Review by jmagman — APPROVED** [review]
Cross-referencing to #39172
LGTM

File: shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm [hunk 1]
```diff
@@ -41,20 +41,6 @@
...
-          task_runners_.GetPlatformTaskRunner()->GetTaskQueueId())) {
-    BOOL isRunningOnMac = NO;
-    if (@available(iOS 14.0, *)) {
-      isRunningOnMac = [NSProcessInfo processInfo].iOSAppOnMac;
-    }
-    if (!isRunningOnMac) {
-      // Pressure tested on iPhone 13 pro, the oldest iPhone that supports refresh rate greater than
-      // 60fps. A flutter app can handle fast scrolling on 80 fps with 6 PlatformViews in the scene
-      // at the same time.
-      new_max_refresh_rate = 80;
```
Comment (hellohuanlin) [inline shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:55]: Ask to learn: Does this mean that we wanted to cap to 80FPS when there's platform view on screen (but if no platform view, it's capped at 120FPS)? By removing this, we remove the 80FPS cap, and let OS to decide what refresh rate is.

Comment (jonahwilliams) [reply shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:55]: When there were 5 or more platform views, we set the cap to 80. Now we always set to whatever the maximum rate is, in this case 120. AFAIK, we don't fully support adjusting the frame rate.

Comment (jmagman) [reply shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm:55]: For background, check out https://github.com/flutter/flutter/issues/116640 https://github.com/flutter/engine/pull/39172 (and https://github.com/flutter/engine/pull/39172), there's a lot of info in that issue.

**Comment by jmagman** [comment]
Made the benchmark slightly more noisy?

https://flutter-flutter-perf.skia.org/e/?queries=sub_result%3D90th_percentile_frame_rasterizer_time_millis%26sub_result%3D99th_percentile_frame_rasterizer_time_millis%26sub_result%3Daverage_frame_rasterizer_time_millis%26sub_result%3Dworst_frame_rasterizer_time_millis%26test%3Dplatform_views_scroll_perf_ad_banners__timeline_summary&selected=commit%3D40025%26name%3D%252Carch%253Dintel%252Cbranch%253Dmaster%252Cconfig%253Ddefault%252Cdevice_type%253DiPhone_11%252Cdevice_version%253Dnone%252Chost_type%253Dmac%252Csub_result%253D90th_percentile_frame_rasterizer_time_millis%252Ctest%253Dplatform_views_scroll_perf_ad_banners__timeline_summary%252C

**Pull request merged**